hail/src/main/scala/is/hail/variant/MatrixTable.scala

```diff
@@ -140,7 +142,9 @@ case class MatrixTableSpec(
 }
 
 object FileFormat {
-  val version: SemanticVersion = SemanticVersion(1, 0, 0)
+  val version: SemanticVersion = SemanticVersion(1, 1, 0)
+
+  val indicesVersion: SemanticVersion = SemanticVersion(1, 1, 0)
```

Comment:
adding *another* version to worry about seems worse than incrementing the minor version. I will push strongly for the latter approach.

Reply:
What happens when we increment again? How do we make sure that `1.1.0` tables are still read appropriately?

The intent of `indicesVersion` (willing to bikeshed on name) is to enshrine in a constant the version in the `1.0` series of file formats where indices appeared. It is used to compare against the TableSpec's version in TableRead.

Reply:
> How do we make sure that 1.1.0 tables are still read appropriately

You mean, what happens when we update the index design? That's a good point, we need to add an IndexSpec that contains the relevant information.

Though now that I'm think about this... Can't the index metadata just be part of the RVDSpec?

Reply:
I don't like that because the RVDSpec is not versioned.

Reply:
it is -- the existing concrete specs are immutable, you can't change OrderedRVDSpec for instance. You could add an IndexedRVDSpec, however, that contains the index metadata!